service/patch.go

```diff
@@ -179,7 +179,7 @@ func (uis *UIServer) schedulePatch(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		ver, err := model.FinalizePatch(projCtx.Patch, &uis.Settings)
+		ver, err := model.FinalizePatch(projCtx.Patch, uis.Settings.Credentials["github"])
```

Comment:
should we be checking if this credential exists? 

maybe it'd be better to put a method on the settings object that we can call and change things behind the scenes: 
- I think the fact that we have a single github token site wide is a risk, and I'd like to be able to have per-project. 
- as we move the config into the database and out of the file, I expect the fact that the credentials are in a map will stop being a thing.

Reply:
added a method as suggested